cache/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "random_id" "suffix" {
```

Comment:
There's no need to create this if `bucket_name` is defined. Create this conditionally.